webrender/src/frame.rs

```diff
@@ -320,6 +320,10 @@ impl Frame {
                                                   background_color,
                                                   self.frame_builder_config);
 
+        if let Some(ref mut old_builder) = self.frame_builder {
```

Comment:
I see. You have a point. I wish we could do `#[derive(Clear)]` to enforce that without manually tracking the new items.